packages/sensors/pubspec.yaml

```diff
@@ -12,6 +12,7 @@ flutter:
     pluginClass: SensorsPlugin
 
 dependencies:
+  async: ^2.0.6
```

Comment:
Is this OK? If not, both platforms expose an accelerometer with gravity removed, but this seemed like the easiest way.